target/linux/ar71xx/base-files/lib/ar71xx.sh

```diff
@@ -504,6 +504,9 @@ ar71xx_board_detect() {
 	*"CF-E320N v2")
 		name="cf-e320n-v2"
 		;;
+	*"CF-E355AC")
```

Comment:
Are you sure that there is only one version of this device? As far as I can see on vendor website, v2 is mentioned in [firmware download section](http://en.comfast.com.cn/download/FirmwareUpgrade/).

Reply:
Hi all, late to the game but there is indeed a v2 of this AP with different QC chips (I have the Access Point at my house now and am trying to get LEDE working). http://en.comfast.com.cn/product/WirelessAP/item-218.html

Reply:
Is it a new model with Wave2? 
the manufacturer assured me that there was only one version of that model.

Reply:
Yes, according to http://en.comfast.com.cn/product/WirelessAP/item-218.html it has Wave2

Reply:
I'm so not surprised about v2 :)